src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -139,7 +134,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(18),
-    color: COLORS.anon_primary,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, please ensure that the color change from `COLORS.anon_primary` to `COLORS.white2` adheres to accessibility standards for contrast ratios.


<!-- This is an auto-generated comment by OSS CodeRabbit -->